base/database/etl_homologene.py

```diff
@@ -59,17 +61,29 @@ def fetch_homologene(homologene_fname: str):
     # First, fetch records with a homolog ID that possesses a C. elegans gene.
     elegans_set = dict([[int(x[0]), x[3]] for x in response_csv if x[1] == '6239'])
```

Comment:
consider declaring a constant for the magic value with a self-documenting name, ie.: 
```
C_ELEGANS_GENE='6239'
elegans_set = dict([[int(x[0]), x[3]] for x in response_csv if x[1] == C_ELEGANS_GENE ])
```

Reply:
side note, #fetch_taxon_ids() function body doesn't do error handling.